<?xml version="1.0" encoding="UTF-8"?>
<bajadoc version="2.0" createdBy="niagara-baja-doclet-1.0.9" createdAt="03-May-2024" createdOn="883e7f7a9875">
<package module="seriesTransform" runtimeProfile="rt" name="javax.baja.seriestransform.exceptions">
<description/>
<class packageName="javax.baja.seriestransform.exceptions" name="ConfigException" category="exception"><description>The Config exception indicates an exception generated when attempting to&#xa; configure a graph node for the transform graph.</description></class>
<class packageName="javax.baja.seriestransform.exceptions" name="SchemaException" category="exception"><description>The Schema exception indicates an exception involving the structure or &#xa; resolving of a schema for a transform graph or graph node.</description></class>
<class packageName="javax.baja.seriestransform.exceptions" name="SourceException" category="exception"><description>A Source exception identifies an exception that occurs when attempting to&#xa; resolve data for the transform graph.</description></class>
<class packageName="javax.baja.seriestransform.exceptions" name="TransformException" category="exception"><description>The Transform Exception is the root exception for checked exceptions in&#xa; the transform graph architecture.</description></class>
</package>
</bajadoc>
